src/devices/nodon.ts

```diff
@@ -103,12 +91,13 @@ const definitions: Definition[] = [
             await reporting.onOff(ep1);
             await reporting.onOff(ep2);
         },
+        ota: ota.zigbeeOTA,
     },
     {
         zigbeeModel: ['SIN-4-2-20_PRO'],
         model: 'SIN-4-2-20_PRO',
         vendor: 'NodOn',
-        description: 'Double LED relay',
+        description: 'Lighting Relay Switch',
```

Comment:
Can you normalize the casing for all descriptions? (e.g. `Lighting relay switch`)